src/sparsezoo/v2/objects/file.py

```diff
@@ -71,29 +78,43 @@ def __init__(
         }
 
     @classmethod
-    def from_dict(cls, file: Dict[str, Any]) -> "File":
+    def from_dict(
+        cls, file: Dict[str, Any], owner_path: Optional[str] = None
+    ) -> "File":
         """
         Factory method for creating a File class object
         from a file dictionary
         (useful when working with the `request_json` from NeuralMagic).
 
         :param file: a dictionary which contains an information about
             the file (as returned by NeuralMagic API)
+        :param owner_path: path of the parent Directory
         :return: File class object
         """
         name = file.get("display_name")
         path = file.get("path")
         url = file.get("url")
 
-        return cls(
-            name=name,
-            path=path,
-            url=url,
-        )
+        return cls(name=name, path=path, url=url, owner_path=owner_path)
+
+    def get_path(self, download_directory: Optional[str] = None):
```

Comment:
This would be a bit ugly in terms of user experience. The objective of `get_path()` is to be seamlessly evoked directly as a method of a class object. Your suggestion implies additionally exporting the `get_path` function, which is too much of a hassle IMO.